library/Init/Data/Stack/Basic.lean

```diff
@@ -0,0 +1,41 @@
+/-
+Copyright (c) 2019 Microsoft Corporation. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Daniel Selsam
+
+Simple stack API implemented using an array.
+-/
+prelude
+import Init.Data.Array
+import Init.Data.Int
+universes u v w
+
+structure Stack (α : Type u) :=
+(vals : Array α := Array.empty)
+
```

Comment:
We can now write `#[]` for `Array.empty`

Reply:
Noted.